.github/workflows/CI.yml

```diff
@@ -41,3 +41,39 @@ jobs:
         run: npx envinfo
       - name: Build
         run: ./configure && make -j8
+  lint-addon-docs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
```

Comment:
```suggestion
    - name: Use Node.js 10
```

There's no matrix, so the version isn't printed:

![image](https://user-images.githubusercontent.com/2352663/72215673-88dda080-3516-11ea-82d6-604bc9bc2dc7.png)

Reply:
This was copied from the existing job definitions in this workflow. I've added a commit to this PR to fix up all of the `Use Node.js` step names.